Workbooks/LogAnalytics Workspace/AtScale/Workspace Insights At Scale.workbook

```diff
@@ -217,7 +219,8 @@
               "formatter": 13,
               "formatOptions": {
                 "linkTarget": "Resource",
```

Comment:
add "subTarget": "insights", here. I noticed that it's not linking directly to the insights. This is because on the resource blade side, the id was called workspaceInsights. For testing purposes, you can use workspaceInsights. But I have already reached out to Omer Ziv to see if he can change the id to 'insights' so it conforms with the rest of our partner teams. Once he makes that change, this will automatically light up

`"subTarget": "insights",`